src/transformers/models/conditional_detr/image_processing_conditional_detr.py

```diff
@@ -332,10 +332,13 @@ def prepare_coco_detection_annotation(
 
     if annotations and "keypoints" in annotations[0]:
         keypoints = [obj["keypoints"] for obj in annotations]
+        # Apply the keep mask here to filter the relevant annotations
+        keypoints = [keypoints[i] for i in range(len(keypoints)) if keep[i]]
```

Comment:
```suggestion
        keypoints = keypoints[keep]
```
this should be the same no?